src/Medidata.ZipkinTracer.Core/Medidata.ZipkinTracer.Core.csproj

```diff
@@ -58,23 +58,34 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
-    <Reference Include="Thrift, Version=0.9.1.3, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Thrift.0.9.1.3\lib\net35\Thrift.dll</HintPath>
-    </Reference>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="ITraceProvider.cs" />
     <Compile Include="ITracerClient.cs" />
     <Compile Include="IZipkinConfig.cs" />
     <Compile Include="Middlewares\ZipkinMiddleware.cs" />
+    <Compile Include="Models\Annotation.cs" />
+    <Compile Include="Models\AnnotationBase.cs" />
+    <Compile Include="Models\BinaryAnnotation.cs" />
+    <Compile Include="Models\Endpoint.cs" />
+    <Compile Include="Models\References\AnnotationType.cs" />
+    <Compile Include="Models\References\ZipkinConstants.cs" />
+    <Compile Include="Models\Serialization\Json\JsonAnnotation.cs" />
```

Comment:
The folder 'Json/' is for the better organization of the files - it won't appear in the namespaces.

In the other hand the naming of the `Json...` classes are following the original OpenZipkin implementation (you can see an example [here](https://github.com/openzipkin/zipkin/blob/master/zipkin-common/src/main/scala/com/twitter/zipkin/json/JsonSpan.scala)). Also, the original (i.e. before the refactoring) serialization classes were named as `Serializable...` so this is just a renaming of those files to be specific for the used serialization here ('Serializable' is too general).

As for the merge, having separate serialization classes is following the original implementation from Bernie (#67), I think his intention - which I also agree with - is to separate the actual models from their representation in serialization; as these are separate concerns it is better to keep them separate for maintainability.